File: aspnet-core/src/HC.WeChat.Core/ExhibitionShops/ExhibitionShop.cs
using Abp.Domain.Entities;
using Abp.Domain.Entities.Auditing;
using HC.WeChat.WechatEnums;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace HC.WeChat.ExhibitionShops
{
    /// <summary>
    /// 陈列店铺
    /// </summary>
    [Table("ExhibitionShops")]
    public class ExhibitionShop : Entity<Guid>
    {

        /// <summary>
        /// 创建时间
        /// </summary>
        [Required]
        public virtual DateTime CreateTime { get; set; }

        /// <summary>
        /// 零售客户Id
        /// </summary>
        public virtual Guid? RetailerId { get; set; }

        /// <summary>
        /// 店铺名称
        /// </summary>
        [Required]
        [StringLength(200)]
        public virtual string ShopName { get; set; }

        /// <summary>
        /// 店铺地址
        /// </summary>
        [StringLength(500)]
        public virtual string ShopAddress { get; set; }

        /// <summary>
        /// 陈列图片
        /// </summary>
        [Required]
        public virtual string PicPath { get; set; }

        /// <summary>
        /// 投票数
        /// </summary>
        public virtual int? Votes { get; set; }

        /// <summary>
        /// 是否通过审核
        /// </summary>
        [Required]
        public virtual int Status { get; set; }

        /// <summary>
        /// 店铺Id
        /// </summary>
        [Required]
        public virtual Guid ShopId { get; set; }

        /// <summary>
        /// 修改时间
        /// </summary>
        public virtual DateTime? AuditTime { get; set; }
    }
}

File: ng-admin/src/app/shared/entity/marketting/exhibitionShop.ts
export class ExhibitionShop implements IExhibitionShop {
    id: string;
    createTime: Date;
    retailerId: string;
    shopName: string;
    shopAddress: string;
    picPath: string;
    votes: number;
    status: number;
    shopId: string;
    auditTime: Date;
    constructor(data?: IExhibitionShop) {
        if (data) {
            for (var property in data) {
                if (data.hasOwnProperty(property))
                    (<any>this)[property] = (<any>data)[property];
            }
        }
    }

    init(data?: any) {
        if (data) {
            this.id = data["id"];
            this.createTime = data["createTime"];
            this.retailerId = data["retailerId"];
            this.shopName = data["shopName"];
            this.shopAddress = data["shopAddress"];
            this.picPath = data["picPath"];
            this.votes = data["votes"];
            this.status = data["status"];
            this.shopId = data["shopId"];
            this.auditTime = data["auditTime"];
        }
    }

    static fromJS(data: any): ExhibitionShop {
        let result = new ExhibitionShop();
        result.init(data);
        return result;
    }

    toJSON(data?: any) {
        data = typeof data === 'object' ? data : {};
        data["id"] = this.id;
        data["createTime"] = this.createTime;
        data["retailerId"] = this.retailerId;
        data["shopName"] = this.shopName;
        data["shopAddress"] = this.shopAddress;
        data["picPath"] = this.picPath;
        data["votes"] = this.votes;
        data["status"] = this.status;
        data["shopId"] = this.shopId;
        data["auditTime"] = this.auditTime;
        return data;
    }

    clone() {
        const json = this.toJSON();
        let result = new ExhibitionShop();
        result.init(json);
        return result;
    }
}
export interface IExhibitionShop {
    id: string;
    createTime: Date;
    retailerId: string;
    shopName: string;
    shopAddress: string;
    picPath: string;
    votes: number;
    status: number;
    shopId: string;
    auditTime: Date;
}